webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml

```diff
@@ -31,9 +31,14 @@
         template="system:/groovy/portal/webui/container/UIContainer.gtmpl"
         cssClass="border-box-sizing">
         <access-permissions>Everyone</access-permissions>
-        <container id="topNewsPage" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
-          <access-permissions>*:/platform/users</access-permissions>
-        </container>
+        <portlet-application>
```

Comment:
I think that we should rework the news template layout, to be verified with the PO

Reply:
I think that the container id="newsPage" is nomore needed, even css classes already added not needed, to be verified with the final look